Package.swift

```diff
@@ -10,19 +10,22 @@ let package = Package(
                 .tvOS(.v12),
                 .watchOS(.v2)],
     products: [
-        .library(name: "ParseObjC", targets: ["ParseCore"]),
+        .library(name: "ParseObjC", targets: ["Parse"]),
```

Comment:
Why this change and is this a breaking change?

It seems a bit odd to name the core module `Parse`, as the term "Parse" is arguably referring to the whole SDK. To me is makes more sense to name the core module `ParseCore` to indicate that it's the basis that is always needed, while others are optional.

Reply:
Hi, yes it is an important change because in SDK target has the name "Parse". When we set "ParseCore" it causes an error downloading.

Reply:
Not sure I understand. Why can't we keep the name `ParseCore` and change the target name? I mean this is only an issue for the LiveQuery module, right?

Reply:
@mtrezza This is my first time using SPM and I didn't know you had to do `import ParseCore;`. I can try to revert it back and fix the downloading error

Reply:
@vazarkevych Could you take a look at this so we can close this PR? It seems to be the last remaining issue here.

Reply:
@mtrezza Hi, This issue with renaming is caused because of we have in one project two code languages (swift, objc-c) and it has two different imports SDK to the project. If we left the target name ParseCore then we get the error with building ParseLiveQuery because the compiler doesn't find "ParseCore" because the target name in the project ParseSDK is "Parse", but SPM should work that SPM imports frameworks from "Package.swift" file. 

And, if we rename the target name from "ParseCore" to "Parse", we have successfully built the project ParseLiveQuery and have works SPM.

We saw two ways to resolve this issue:
First - Left the target name "Parse" (All our users will require to change the name of the import to "Parse" from "ParseCore" only in Swift, on Objective-c no changes are required).
Second - We need to rename the project "ParseSDK" targets from "Parse" to "ParseCore"

Reply:
> Second - We need to rename the project "ParseSDK" targets from "Parse" to "ParseCore"

Let's see if I get this right. If we do this, then developers won't have a rename their `ParseCore` imports to `Parse`. If there are no other consequences then this is the preferred way.

Reply:
Any updates on this?

Reply:
We just need someone to do this renaming, then we should be good to merge.

Reply:
@VolodyaNazarkevych @mtrezza  Either of you want to take this over?

Reply:
It's up for grabs!

Reply:
@mtrezza Can you commit 1 example of tedious work? I'll finish the rest. I don't quite understand what needs to be changed

Reply:
See https://github.com/parse-community/Parse-SDK-iOS-OSX/pull/1712#discussion_r1175802198.

Reply:
Any updates on this? 
Is this repo not really supported anymore? I'll adjust my expectations if so

Reply:
You can see the status from the message above.

Reply:
@dplewis Do you want to take a look at this, we could add an additional bounty.

Reply:
@dplewis Do you want to take a look at this, we could add an additional bounty.